modules/mobile/app/serializers/mobile/v0/claims_and_appeals_overview_serializer.rb

```diff
@@ -0,0 +1,57 @@
+# frozen_string_literal: true
+
+require 'ostruct'
+
+module Mobile
+  module V0
+    class ClaimsAndAppealsOverviewSerializer
```

Comment:
Could this top level serializer be an adapter instead? 

We're bending Fast JSON API in unnatural ways probably not intended by its authors 😀 

I think we might want an intermediary step that gets a claim or appeal into the right shape before passing it to a `Mobile::V0::[Appeal|Claim]OverviewSerializer`. At the top level it might be as simple as doing  `render json:, { data: formatted_entries }.to_json`.

Or since Appeal and Claim Overview Serializers are so similar you might be able to have one serializer that can dynamically set it's type and then you could use the [regular collection serialization method](https://github.com/Netflix/fast_jsonapi#collection-serialization).